packages/local/test/commands/local/index.test.ts

```diff
@@ -24,33 +26,38 @@ describe('local', () => {
   describe('without arguments', function () {
     describe('without flags', function () {
       test
-      .stub(procfile, 'loadProc', function (procfile: any) {
-        expect(procfile).is.equal('Procfile', 'it defaults to loading `Procfile`')
+      .stub(procfile, 'loadProc', function () {
+        // eslint-disable-next-line prefer-rest-params
+        expect(arguments[0]).is.equal('Procfile', 'it defaults to loading `Procfile`')
```

Comment:
This is a questionable solution to an obnoxious problem. Context here: https://salesforce-internal.slack.com/archives/C048U82QVKQ/p1667949410523679

This was the least terrible approach I could come up with.